lib/astarte/client/credentials.ex

```diff
@@ -135,33 +149,77 @@ defmodule Astarte.Client.Credentials do
   end
 
   def set_expiry(%Credentials{} = credentials, expiry_seconds)
```

Comment:
Note to ourself: should we maybe take `{expiry_seconds, :second}` instead of a bare number?